Applications/VLMEval/README.md

```diff
@@ -0,0 +1,50 @@
+#  VLMEval
+
+An example that:
+
+- downloads a vision language model (Qwen-VL-2B)
+- processes an image with a prompt
+
+> Note: this _must_ be built Release, otherwise you will encounter
+stack overflows.
```

Comment:
This part is no longer true -- I guess we should update the LLMEval about that too.  Debug works fine, though slower :-)

Reply:
I removed from both VLMEval and LLMEval!